pkg/apis/types/types.go

```diff
@@ -104,3 +104,12 @@ const (
 	//TotalWaitTime is the max time duration to wait for doing snapshot operation on all the replicas
 	TotalWaitTime = 60
 )
+
+const (
+	// OpenEBSDisableDependantsReconcileKey is the annotation key that decides to create
+	// children objects with OpenEBSDisableReconcileKey as true or false
+	OpenEBSDisableDependantsReconcileKey = "reconcile.openebs.io/disable-dependants"
+
+	// OldPoolName is the old pool that needs to be imported and renamed
+	OldPoolName = "cstorpoolinstance.openebs.io/oldname"
```

Comment:
n_: OldPoolName => CSPIOldPoolName
n_: cstorpoolinstance.openebs.io/oldname => backwardcompatibility.openebs.io/cspi-v1alpha-name

Reply:
Can we have `backwardcompatibility.openebs.io/csp-v1alpha-name` instead as it migrating from csp to cspi?

Reply:
IMO -- we are tying the annotation specific to some of the things that we are trying to do. Like old/new/backward etc..

I am thinking on these lines: 
CSPI has a feature that gives you the capability to import the pool. Now this import requires the name of the pool and the system knows the name of the pool to import in most cases. Thre are certain scenarios where this name needs to be overridden and CSPI has that feature also to do that. So if you provide a specific annotation on CSPI -- the value of that specific annotation will be used as the pool name to run the import command.

Reply:
Yes, the annotation can be used for not only migration or but for new features as well. Like detaching disks from one setup having a CSPC and attaching to another setup and just creating a new CSPC and pool import should happen easily. So having `cstorpoolinstance.openebs.io/oldname` makes more sense.

Reply:
Yes @sonasingh46, I am also thinking in the same lines. Attaching [slack thread](https://openebs-community.slack.com/archives/C3NPGQ6G3/p1590054282279800) will elaborate more on the use case.
While coming to the naming convention `cstorpoolinstance.openebs.io/old-pool-name` or `cstorpoolinstance.openebs.io/pool-name`.